src/components/dar_table/DarElectionRecords.js

```diff
@@ -73,16 +75,18 @@ export default function DarElectionRecords(props) {
       div({style: Object.assign({}, Styles.TABLE.SUBMISSION_DATE_CELL, recordTextStyle)}, [getElectionDate(election)]),
       div({style: Object.assign({}, Styles.TABLE.DAC_CELL, recordTextStyle)}, [dac ? dac.name : '- -']),
       electionStatusTemplate(consoleType, dar, election, recordTextStyle, votes, showVotes, history),
-      h(DarTableActions, {
-        baseStyle: Object.assign({}, Styles.TABLE.ELECTION_ACTIONS_CELL, recordTextStyle),
-        updateLists,
-        openConfirmation,
-        history,
-        electionInfo,
-        index,
-        consoleType,
-        extraOptions
-      })
+      div({isRendered: consoleType !== consoleTypes.SIGNING_OFFICIAL}, [
```

Comment:
Adding the `div` here causes the table to break in other pages

<img width="1752" alt="Screen Shot 2021-07-21 at 9 15 52 AM" src="https://user-images.githubusercontent.com/6664778/126494824-a2d45dba-d1e9-47dc-b8a0-d2be0a92f917.png">

You can add the `isRendered` condition in the config object for `DarTableActions` instead